src/lib/contentful/types/Article.ts

```diff
@@ -1,16 +1,48 @@
+import type { Document } from "@contentful/rich-text-types";
+
 export type Article = {
 	slug: string;
 	title: string;
 	ingress: string;
-	body?: {
-		json: any;
+	body: {
+		json: Document;
+		links: ArticleLinks;
 	};
-	articleHeroImage: {
+	articleHeroImage?: {
 		title: string;
 		url: string;
-	};
+	} | null;
```

Comment:
The ` | null ` here seems superfluous as you've configured as `articleHeroImage? : { }`

Reply:
Ok, remove the `?`